rest_sensors.cpp

```diff
@@ -341,6 +341,10 @@ int DeRestPluginPrivate::createSensor(const ApiRequest &req, ApiResponse &rsp)
         sensor.setSwVersion(map[QLatin1String("swversion")].toString());
         sensor.setType(type);
 
+        sensor.removeItem(RConfigReachable);
```

Comment:
Interesting, I'm not sure if `RConfigReachable` is used in some CLIP sensors. It is allowed by the POST REST API call.

Reply:
Double-checked: the Hue bridge shows `config/reachable` for CLIP sensors.  I'll ditch this line.